plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/FluidAttacksReader.java

```diff
@@ -48,7 +62,14 @@ public TestSuiteResults parse(ResultFile resultFile) throws Exception {
         for (CSVRecord record : records) {
             TestCaseResult testCaseResult = new TestCaseResult();
             // Read only useful rows of the csv results
-            if (record.get("description").split("OWASP").length < 2) {
+            String description;
+            try {
```

Comment:
I prefer not to use Exceptions in a regular programming way. I had a quick (untested) look to `CSVRecord` class. Can you have a look at the method `isMapped` as a replacement of the try/catch?

Reply:
Will do! Thanks :)